### PR TITLE
Add HOKK logo and update info

### PR DESCRIPTION
**Pull request by HokkaiduInu**
This pull request may seem odd since this is the former contract address of HOKK Finance (https://www.coingecko.com/en/coins/hokk-finance?locale=en) token prior to their migration. We will clarify on that.

HOKK Finance represents the second version contract of this token developed by the creators after they decided to discontinue the original project, hence why this token was abandoned. This move from the creators was ill-intentioned, aimed at extracting more financial gains from the original HOKK holders. HOKK's market capitalization suffered greatly because of this.

We are a community of volunteers that is working to revive the original HOKK. If you are not aware, this token was one of the original dog coins when the meme space was popular back in 2021, rivaling SHIB during the time. In fact, HOKK's market cap was higher than SHIB's at one point, reaching a $822M market cap in early 2021. Just over a week ago, the market cap of HOKK was $40k. We have reached $2M market cap in under a week, and is currently at $1.2M. We have been experiencing healthy sustained volume for some time now. We are gaining a lot of traction on all social media and even have top SHIB holders investing in HOKK, so we are definitely on many people's radar.

Just for proof if needed, here are some HOKK buys from SHIB whales, including the former #1 SHIB holder (first 3 links)
https://etherscan.io/tx/0x07c5310ad64d990d673faf0906fbe97ee43f6b0ad027e8d146ecbec5975ad96a
https://etherscan.io/tx/0x0103bd7f039523dd2417b822c9916cd1df23d36f8fdd33572e46f59b60a3172a
https://etherscan.io/tx/0x230d8f70471acabd7561c3ad38b1c41de577e8b11817756d0161595358cc4683
https://etherscan.io/tx/0x50f4597ca77125caa4da1d34114cc1729ab8cc313a8b8fb79e2a0033235606b1
https://etherscan.io/tx/0xac3c78b1933f5fa4c86ad265084d9d677346c9462847e194807abfb2a8d435c9

We are now on CoinGecko (https://www.coingecko.com/en/coins/hokkaido-inu-2) and will be on CoinMarketCap soon, so updating the info in trustwallet will be extremely helpful.

If you have any questions, feel free to hop in our telegram, https://t.me/HOKKED_AND_LOADED. We are very active there. Our twitter is @HokkaInuEth.

Thank you very much,
The HOKK Community - Hokkaidu Inu

### CODE REVIEW

**Comment by Cryptocool1** [comment]
Hello, what is the purpose of adding the old contract address?

**Comment by HokkaiduInu** [comment]
Hey Crytocool1, 

You can read the comment above where I explain everything. But just to recap, we are a group of volunteers working to revive this coin. HOKK was one of the biggest meme coins back in 2021, beating SHIB and had a ATH of $822M before the devs migrated to a new contract. In under 10 days, we brought the market cap from $40k to over $2M. We are currently at $1.2M. We are active on social media and are gaining lots of traction. Several SHIB whales have invested in us, so we are definitely on many people's radar.

Since the devs have abandoned this token and migrated to a new contract, the community has been working to revive the old contract, and thus we are trying to update all information for this coin. We are back on CoinGecko and will be on CoinMarketCap soon. Our twitter has been gaining lots of traction recently since many people remember HOKK back in 2021. 

Please read the original comment in the pull request to get an overview.

**Comment by Cryptocool1** [comment]
Hello, For returning to Trust wallet you should do new audit from reputable companies and be listed on CMC

**Comment by HokkaiduInu** [comment]
Hello,

We have already been audited by techrate.org as seen here: https://github.com/TechRate/Smart-Contract-Audits/blob/main/2018-21%20A-M/Hokkaidu%20Inu.pdf

This audit by techrate.org was done on the April 23rd, 2021. This contract was already renounced by then on April 16th, 2021. Seeing as this contract was abandoned and not a single thing has changed about contract because it was renounced, we do not think another audit is necessary. Can an exception be made for this case?

Thank you very much for your time. We look forward to hearing back from you!

**Comment by Cryptocool1** [comment]
Please send your CMC link

**Comment by HokkaiduInuToken** [comment]
Hello,

We have applied for CMC, but we are not yet listed on the mainsite, I will
leave a link to CMC's Dexscan however

https://coinmarketcap.com/dexscan/ethereum/0x9314941c11d6dee1d7bf93113eb74d4718949f3b/



Best Wishes,

Hokkaidu Inu Community

**Comment by HokkaiduInuToken** [comment]
> Please send your CMC link

https://coinmarketcap.com/dexscan/ethereum/0x9314941c11d6dee1d7bf93113eb74d4718949f3b/

Hello friend, CMC was applied around 1 week ago but we are still awaiting to be listed there, I will leave my ID request number here and the CMC-Dexscan for HOKK/Weth here in case that can be of use

''Your request (848587) has been updated.''

**Comment by Bspring305** [comment]
> This pull request may seem odd since this is the former contract address of HOKK Finance (https://www.coingecko.com/en/coins/hokk-finance?locale=en) token prior to their migration. We will clarify on that.
> 
> HOKK Finance represents the second version contract of this token developed by the creators after they decided to discontinue the original project, hence why this token was abandoned. This move from the creators was ill-intentioned, aimed at extracting more financial gains from the original HOKK holders. HOKK's market capitalization suffered greatly because of this.
> 
> We are a community of volunteers that is working to revive the original HOKK. If you are not aware, this token was one of the original dog coins when the meme space was popular back in 2021, rivaling SHIB during the time. In fact, HOKK's market cap was higher than SHIB's at one point, reaching a $822M market cap in early 2021. Just over a week ago, the market cap of HOKK was $40k. We have reached $2M market cap in under a week, and is currently at $1.2M. We have been experiencing healthy sustained volume for some time now. We are gaining a lot of traction on all social media and even have top SHIB holders investing in HOKK, so we are definitely on many people's radar.
> 
> Just for proof if needed, here are some HOKK buys from SHIB whales, including the former #1 SHIB holder (first 3 links) https://etherscan.io/tx/0x07c5310ad64d990d673faf0906fbe97ee43f6b0ad027e8d146ecbec5975ad96a https://etherscan.io/tx/0x0103bd7f039523dd2417b822c9916cd1df23d36f8fdd33572e46f59b60a3172a https://etherscan.io/tx/0x230d8f70471acabd7561c3ad38b1c41de577e8b11817756d0161595358cc4683 https://etherscan.io/tx/0x50f4597ca77125caa4da1d34114cc1729ab8cc313a8b8fb79e2a0033235606b1 https://etherscan.io/tx/0xac3c78b1933f5fa4c86ad265084d9d677346c9462847e194807abfb2a8d435c9
> 
> We are now on CoinGecko (https://www.coingecko.com/en/coins/hokkaido-inu-2) and will be on CoinMarketCap soon, so updating the info in trustwallet will be extremely helpful.
> 
> If you have any questions, feel free to hop in our telegram, https://t.me/HOKKED_AND_LOADED. We are very active there. Our twitter is @HokkaInuEth.
> 
> Thank you very much, The HOKK Community - Hokkaidu Inu

thanks so much

File: blockchains/ethereum/assets/0xC40AF1E4fEcFA05Ce6BAb79DcD8B373d2E436c4E/info.json [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+        "memes"
+    ],
+    "links": [
+        {
+            "name": "twitter",
+            "url": "https://twitter.com/HokkaInuEth"
+        },
+        {
+            "name": "telegram",
+            "url": "https://t.me/HOKKED_AND_LOADED"
```
Comment (HokkaiduInuToken) [inline blockchains/ethereum/assets/0xC40AF1E4fEcFA05Ce6BAb79DcD8B373d2E436c4E/info.json:21]: This link needs to be updated to ''url'': ''https://t.me/HokkaiduInuOfficial''

**Review by HokkaiduInuToken — COMMENTED** [review]
This telegram link needs to be updated to the new one, as HOKKED_AND_LOADED no longer exists.

**Comment by HokkaiduInuToken** [comment]
> Please send your CMC link

We are currently awaiting on CMC listing, we applied around 1 week ago. We do not know how long it would take. I trust our payment is safe until then so we can proceed as soon as CMC goes ahead?

**Comment by Cryptocool1** [comment]
Please come back to us once you update your token on CoinMarketCap

**Comment by HokkaiduInuToken** [comment]
Hey friend, did you guys go live with the request? We needed to change the
logo before committing, is there anywah we could still do this?

**Comment by HokkaiduInuToken** [comment]
We are now back in Coinmarket cap

https://coinmarketcap.com/currencies/hokkaidu-inu/


However, the team member responsible for the github account that initiated
the request is off.

Could we please update the logo to this?

https://hokkaidoinu.biz/HOKK-200-200.png

Which is the same as CMC


Best Regards,

Hokkaidu Inu community

**Comment by HokkaiduInu** [comment]
Please reopen this pull request. We are now listed on CMC

https://coinmarketcap.com/currencies/hokkaidu-inu/

**Comment by ukvc** [comment]
> Please come back to us once you update your token on CoinMarketCap

Hi, Hokkaidu Inu is now on CoinMarketCap https://coinmarketcap.com/currencies/hokkaidu-inu/ please reopen this pull request, thank you.

**Comment by ukvc** [comment]
> Please send your CMC link

https://coinmarketcap.com/currencies/hokkaidu-inu/

**Comment by ukvc** [comment]
> Please come back to us once you update your token on CoinMarketCap

Hokkaidu Inu is now on CoinMarketCap, please reopen this pull request, thank you

**Comment by ukvc** [comment]
> Please reopen this pull request. We are now listed on CMC
> 
> https://coinmarketcap.com/currencies/hokkaidu-inu/

@Cryptocool1

**Comment by ukvc** [comment]
> Please send your CMC link

https://coinmarketcap.com/currencies/hokkaidu-inu/

**Comment by ukvc** [comment]
https://coinmarketcap.com/currencies/hokkaidu-inu/

**Pull request merged**